tests/Kernel/Paragraphs/ParagraphsTest.php

```diff
@@ -166,26 +168,18 @@ public function testListItem(): void {
     $html = $this->renderParagraph($paragraph);
     $crawler = new Crawler($html);
 
-    $this->assertCount(1, $crawler->filter('.ecl-list-item'));
-    $this->assertEquals('Item title', trim($crawler->filter('.ecl-list-item__title')->text()));
-    $this->assertEquals('Item description', trim($crawler->filter('.ecl-list-item__detail')->text()));
+    $this->assertCount(1, $crawler->filter('article.ecl-content-item'));
+    $this->assertEquals('Item title', trim($crawler->filter('article.ecl-content-item div.ecl-content-item__title')->text()));
+    $this->assertEquals('Item description', trim($crawler->filter('article.ecl-content-item div.ecl-content-item__description')->text()));
 
-    $link_element = $crawler->filter('.ecl-list-item__link');
+    $link_element = $crawler->filter('article.ecl-content-item div.ecl-content-item__title a.ecl-link');
     $this->assertCount(1, $link_element);
     $this->assertEquals('http://www.example.com/', $link_element->attr('href'));
 
-    $meta_elements = $crawler->filter('.ecl-meta__item');
-    $this->assertCount(3, $meta_elements);
-    $this->assertEquals('Meta 1', trim($meta_elements->getNode(0)->nodeValue));
-    $this->assertEquals('Meta 2', trim($meta_elements->getNode(1)->nodeValue));
-    $this->assertEquals('Meta 3', trim($meta_elements->getNode(2)->nodeValue));
+    $this->assertEquals('Meta 1 | Meta 2 | Meta 3', trim($crawler->filter('article.ecl-content-item div.ecl-content-item__meta')->text()));
 
     // No images should be rendered in this variant.
-    $this->assertCount(0, $crawler->filter('img.ecl-image'));
-    // No date should be rendered neither.
-    $this->assertCount(0, $crawler->filter('.ecl-date-block__week-day'));
-    $this->assertCount(0, $crawler->filter('.ecl-date-block__day'));
-    $this->assertCount(0, $crawler->filter('.ecl-date-block__month'));
+    $this->assertCount(0, $crawler->filter('article.ecl-content-item > div.ecl-u-d-lg-block'));
```

Comment:
Can we use `'article.ecl-content-item > div[role="img"].ecl-u-d-lg-block'` as it's done below? So it's more clear we are referring to the image part.

Reply:
I prefer to use `$this->assertCount(0, $crawler->filter('article.ecl-content-item > div[role="img"]'));` because it's more generic and handles 2 cases:
- no image at all
- no image visible

Reply:
Ok that would work too.

Reply:
Only, how we test if the image is on the left or on the right? Did we test that before? If we did we should test this also now.

Reply:
I added tests